addons/packages/contour/1.20.1/README.md

```diff
@@ -48,11 +47,116 @@ The following tables shows the providers this package can work with.
+| `envoy.logLevel` | `info` | The Envoy log level. Valid values are `trace`, `debug`, `info`, `warn`, `error`, `critical`, and `off`. |
+| `certificates.useCertManager` | `false` | Whether to use cert-manager to provision TLS certificates for securing communication between Contour and Envoy. If false, the upstream Contour certgen job will be used to provision certificates. If true, the `cert-manager` addon must be installed in the cluster. |
+| `certificates.duration` | `8760h` |  If using cert-manager, how long the certificates should be valid for. If useCertManager is false, this field is ignored. |
+| `certificates.renewBefore` | `360h` |  If using cert-manager, how long before expiration the certificates should be renewed. If useCertManager is false, this field is ignored. |
+
+### Application configuration values
+
+Within the data values file, the `contour.configFileContents` field may optionally contain YAML to put directly in the Contour config file.
+See [the Contour configuration documentation](https://projectcontour.io/docs/v1.20.1/configuration/#configuration-file) for full details on the available options.
+
+An example data values file that specifies this field looks like:
+
+```yaml
+contour:
+  configFileContents:
+    accesslog-format: json
+```
+
+#### Multi-cloud configuration steps
+
+If deploying Contour to **AWS**, you may optionally configure the package to provision a Network Load Balancer (NLB) instead of the default Classic Load Balancer by providing the following data values:
+
+```yaml
+infrastructureProvider: aws
+envoy:
+  service:
+    aws:
+      loadBalancerType: nlb
+```
+
+You may specify platform-specific annotations to be added to the Envoy service by providing the following data values:
+
+```yaml
+envoy:
+  service:
+    annotations:
+      annotation-key-1: val-1
+      ...
+```
+
+## What This Package Does
+
+This package installs Contour, an ingress controller, to enable the user to define rules for routing external traffic to services running inside the cluster.
+Contour supports both the [Ingress API](https://kubernetes.io/docs/concepts/services-networking/ingress/) and the Contour-specific [HTTPProxy API](https://projectcontour.io/docs/v1.20.1/config/fundamentals/) for configuring traffic routing rules.
+
+## Components
+
+* Contour controller
+* Envoy reverse proxy and load balancer
+
+### Supported Providers
+
+The following tables shows the providers this package can work with.
+
+| AWS  |  Azure  | vSphere  | Docker |
+|:---:|:---:|:---:|:---:|
+| ✅  |  ✅  | ✅  | ✅  |
+
+
+## Files
+
+The Contour package optionally accepts a data values file to customize the package's behavior.
+See [Package configuration values](#package-configuration-values) for details of the configurable settings.
+
+An example of the data values file looks like:
+
+```yaml
+namespace: my-custom-contour-namespace
+contour:
+  replicas: 3
+envoy:
+  logLevel: debug
+```
```

Comment:
This section seems pretty redundant given other content, personally I would drop it but it's in the template so I have it here for now.

Reply:
Hey @skriss, totally cool to say "see other sections in this documentation" or the like to avoid redundancies while keeping the section for consistency/in case users look for it.

Reply:
Alright, I think as-is is OK then since it links to the prev section for details and includes a brief example.